### PR TITLE
fix(config): clarify ZEN15 param 27 options

### PR DESCRIPTION
**Pull request by AlCalzone**
in response to https://github.com/zwave-js/node-zwave-js/pull/3768#issuecomment-974891169

### CODE REVIEW

**Comment by AlCalzone** [comment]
@GrizzlyAK I think this is better than the extra verbose labels and still capture the gist of whats going on.
What do you think?

File: packages/config/config/devices/0x027a/zen15.json [hunk 1]
```diff
@@ -136,15 +137,15 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Always On",
+					"label": "When device is plugged in, off otherwise",
```
Comment (GrizzlyAK) [inline packages/config/config/devices/0x027a/zen15.json]: I think some might take "device" to mean either the ZEN15 itself, or something plugged into it, given we are talking about power consumption  of what is plugged _into_ it. I actually think "Always On" here is OK.

File: packages/config/config/devices/0x027a/zen15.json [hunk 2]
```diff
@@ -136,15 +137,15 @@
...
 					"value": 0
 				},
 				{
-					"label": "On when switch is on; Off when switch is off",
+					"label": "When switch is on, off when switch is off",
```
Comment (GrizzlyAK) [inline packages/config/config/devices/0x027a/zen15.json]: I personally prefer the "on" being included here, as it was.

File: packages/config/config/devices/0x027a/zen15.json [hunk 3]
```diff
@@ -136,15 +137,15 @@
...
 					"value": 1
 				},
 				{
-					"label": "Momentary (at status change)",
+					"label": "5 seconds on toggle, off otherwise",
 					"value": 2
```
Comment (GrizzlyAK) [inline packages/config/config/devices/0x027a/zen15.json:149]: This is OK. You might also consider "5 seconds following switch activation, then off."

**Review by GrizzlyAK — COMMENTED** [review]
These are my thoughts. Adding the main description helped to clarify things greatly, so that the shorter options didn't feel so truncated.

**Pull request merged**